dask_geopandas/core.py

```diff
@@ -88,7 +92,19 @@ def crs(self):
     @crs.setter
     def crs(self, value):
         """Sets the value of the crs"""
-        raise NotImplementedError
+        self.dask = self.set_crs(value).dask
+
+    def set_crs(self, value):
+        self._meta.crs = value
+
+        def set_crs(df, crs):
+            df = df.copy(deep=False)
+            df.crs = crs
+            return df
+
+        return self.map_partitions(
+            set_crs, value, meta=self._meta, enforce_metadata=False
+        )
```

Comment:
What happens if you don't supply `_meta` here?  Ideally we would then call `set_crs` to `self._meta` which would be good because then `self.crs` (which presumably calls `self._meta.crs`) would have the right coordinate metadata.